### PR TITLE
feat(components): Toggle renders checkbox

### PR DESCRIPTION
**Pull request by sdwvit**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Toggle is basically a checkbox, but without an option to render as checkbox

**What is the new behavior?**
Exposed classnames parameter allowing to define either it should be checkbox or toggle,
also exported Checkbox component as a wrapper around Toggle

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://273.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://273.talend.surge.sh)

**Comment by sdwvit** [comment]
it can be array of classNames, making it some weird looking checkbox if you like

Fabien has requested classNames customization recently @frassinier

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://273.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://273.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://273.talend.surge.sh)

**Comment by acateland** [comment]
What im thinking,

if we need to display a simple and plain checkbox we shouldn't rely on this component but render a good old, same if you need something that hasn't been described on frontify
```
<input type="checkbox" />
```

**Comment by sdwvit** [comment]
@acateland I thought we decided to always use component

**Comment by jmfrancois** [comment]
please classes => className

File: packages/components/stories/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import './Breadcrumbs';
 import './CircularProgress';
 import './CollapsiblePanel';
 import './ConfirmDialog';
+import './Checkbox';
```
Comment (jmfrancois) [inline packages/components/stories/index.js]: This will not work AFAIK ?

Comment (sdwvit) [inline packages/components/stories/index.js]: it should
packages/components/stories/Checkbox.js

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://273.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://273.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://273.talend.surge.sh)

**Pull request merged**